### PR TITLE
Proxy api.imgur.com and imgur.com

### PR DESCRIPTION
**Pull request by jonleung**

### CODE REVIEW

**Comment by zachlatta** [comment]
Heya! Thanks for the pull request.

So the current layout of Surrogate is a bit confusing right now (sorry about that!). To get this pull request working, you'll want to copy and paste lines 5 - 7 in `conf.d/api.imgur.conf` and `conf.d/imgur.conf` and put them under line 7 in `conf.d/twilio.conf`. You'll also want to change both mentions of `http://` in `conf.d/api.imgur.conf` and `conf.d/imgur.conf` to `https://` :-).

**Comment by jonleung** [comment]
@zachlatta just made the changes that you said verbatim! I don't really have the physical energy to update this again (being sick and all), if you could do me a favor and fix anything that is not right with these configuration files and merge them in, I'd really appreciate that.

Thanks!
Jonathan

File: conf.d/twilio.conf [hunk 1]
```diff
@@ -5,4 +5,10 @@ server {
   location /api.twilio.com/ {
     proxy_pass https://api.twilio.com/;
   }
+  location /api.imgur.com/ {
+    proxy_pass https://api.imgur.com/;
+  }
+  location /imgur.com/ {
```
Comment (zachlatta) [inline conf.d/twilio.conf:11]: Can you add a new line between lines 10 and 11?

File: conf.d/imgur.conf [hunk 2]
```diff
@@ -0,0 +1,8 @@
+server {
+  listen 80;
+  server_name imgur.com;
+
+  location /imgur.com/ {
+    proxy_pass https://imgur.com/;
+  }
+}
```
Comment (zachlatta) [inline conf.d/imgur.conf:8]: Can you delete this file?

File: conf.d/api.imgur.conf [hunk 3]
```diff
@@ -0,0 +1,8 @@
+server {
+  listen 80;
+  server_name api.imgur.com;
+
+  location /api.imgur.com/ {
+    proxy_pass https://api.imgur.com/;
+  }
+}
```
Comment (zachlatta) [inline conf.d/api.imgur.conf:8]: Can you delete this file?

File: conf.d/twilio.conf [hunk 4]
```diff
@@ -5,4 +5,10 @@ server {
   location /api.twilio.com/ {
     proxy_pass https://api.twilio.com/;
   }
+  location /api.imgur.com/ {
```
Comment (zachlatta) [inline conf.d/twilio.conf:8]: And between lines 7 and 8 here?

**Comment by jonleung** [comment]
Closing in favor of #2

**Pull request closed without merging**